examples/cell_tracking.py

```diff
@@ -0,0 +1,175 @@
+"""
+Updated Cell tracking example
+==============================
+
+This example demonstrates how the tracking layer is used
+for displaying 2D + time dataset of cells with colored properties for track-id.
```

Comment:
```suggestion
This example demonstrates how the Track layer can be used for
visualizing cell tracking data, by displaying a 2D + time dataset of 
cells with colored properties for track-id.
```
Maybe something more like this?